### ISSUE TITLE
[cPP Correction] FIA_UAU.7 Protected Authentication feedback

### ISSUE THREAD

**Issue by The-Fiona** [ND-iTC/Documents#374]
**Provide the location of the issue**
cPP B.4.3.1.

**What is the correction request for the cPP? Please describe.**
The cPP allows both remote and local administration to occur. (eg in FMT_SMF.1 selections)

So why is FIA_UA7.1 refined to only be applicable to local administrative users? Surely shoulder surfing can occur remotely too?

**Describe the solution you'd like**
Revert FIA_UAU.7 to be closer to the definition in CC part 2, removing the cPP text "at the local console".
So it would read something like..
"FIA_UAU.7.1 The TSF shall provide only obscured feedback to the **administrative** user while the authentication is in progress."

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by mclearn** [comment]
I have always believed it is because the TOE is actually able to enforce the requirement only at the local console (by preventing feedback or by echoing a masking character). Remote consoles use remote clients and the TOE cannot control/enforce the behaviour of the remote client with respect to visibility at the time of credential entry.

**Comment by arubadean** [comment]
I agree with @mclearn. whether the password is echoed back when accessing the TOE via SSH or the web is going to depend on browser or terminal settings, not anything the TOE does. This change wouldn't really be enforceable as it's outside the scope of the TOE boundary.

**Comment by The-Fiona** [comment]
Thank you for the explanations. I am grateful.
So, it would seem that no change would be the best solution.
@kr15tyk  Can we close this issue please?
